### PR TITLE
Create Climate Setting Flow: Setting Set Points and Mode

### PR DESCRIPTION
**Pull request by Andrewjeska**
https://github.com/seamapi/react/assets/11382084/eac73757-86c6-4aa4-8ce4-c130c6d954fb

### CODE REVIEW

File: src/lib/ui/ClimateSettingForm/ClimateSettingScheduleFormClimateSetting.tsx
Comment (razor-x) [inline src/lib/ui/ClimateSettingForm/ClimateSettingScheduleFormClimateSetting.tsx:59]: Cannot use inline styles in this project.

**Comment by razor-x** [comment]
Let's hold off on merging this until this PR gets in: https://github.com/seamapi/react/pull/533

File: src/lib/ui/ClimateSettingForm/ClimateSettingScheduleForm.tsx
Comment (razor-x) [inline src/lib/ui/ClimateSettingForm/ClimateSettingScheduleForm.tsx:64]: Interesting, what happens if this is not type cast?
```suggestion
      hvacModeSetting: 'heat_cool',
```

File: src/lib/ui/ClimateSettingForm/ClimateSettingScheduleFormClimateSetting.tsx
Comment (razor-x) [inline src/lib/ui/ClimateSettingForm/ClimateSettingScheduleFormClimateSetting.tsx:77]: This nesting is getting a little deep to grok. Consider making the stuff inside `<div className='seam-content'>` its own component (inside the same file)

File: src/lib/ui/ClimateSettingForm/ClimateSettingScheduleFormClimateSetting.tsx
Comment (razor-x) [inline src/lib/ui/ClimateSettingForm/ClimateSettingScheduleFormClimateSetting.tsx:89]: Should these values come from the device / form? Or are these just defaults?

**Review by razor-x — APPROVED** [review]
Looking good, just have a few minor questions.

**Comment by Andrewjeska** [comment]
Would be good to incorporate https://github.com/seamapi/react/pull/541 here before merge

**Pull request merged**
